elytron/src/main/java/org/wildfly/extension/elytron/CredentialStoreResourceDefinition.java

```diff
@@ -192,7 +216,17 @@ private CredentialStoreAddHandler() {
         protected void performRuntime(OperationContext context, ModelNode operation, Resource resource) throws OperationFailedException {
 
             ModelNode model = resource.getModel();
-            String uri = asStringIfDefined(context, URI, model);
+            String location = asStringIfDefined(context, LOCATION, model);
+            boolean modifiable = asBooleanIfDefined(context, MODIFIABLE, model);
+            boolean create = asBooleanIfDefined(context, CREATE, model);
```

Comment:
Here `false` is correctly used.
I think I would just do here and above 
MODIFIABLE.resolveModelAttribute(context, model).asBoolean(MODIFIABLE.getDefaultValue().asBoolean());
CREATE.resolveModelAttribute(context, model).asBoolean(CREATE.getDefaultValue().asBoolean());